### PR TITLE
update README with pg-docker and fix git and env vars

### PR DESCRIPTION
**Pull request by diminator**
### Actual Behaviour
- README instructions had errors instead of successful start of services and dev mode

```bash
# yarn install deps
# error: use node 10

# run relayer
# error: NodeProvider undefined
# update .env.local with extra vars
NETWORK=alfajores
NODE_PROVIDER_TYPE=http
NODE_URL=https://alfajores-forno.celo-testnet.org
# run postgres
docker run --rm  --name pg-docker -e POSTGRES_PASSWORD=docker -d -p 5432:5432 -v $HOME/docker/volumes/postgres:/var/lib/postgresql/data  postgres
# set correct vars ie
DB_USERNAME='postgres'
DB_PASSWORD='docker'
DB_DATABASE='postgres'

# run onboarding 
# error: port 3000 
# need to run load balancer first
docker-compose -f docker-compose.proxy.yml up
# error: Couldn't find the binary git
# update docker-compose.proxy.yml 
RUN apt update && apt install -y git

# run docker-compose -f docker-compose.proxy.yml
# error: Failed to get address for MetaTransactionWalletDeployer from the Registry (also on yarn run:dev onboarding)
# error: connect ECONNREFUSED 127.0.0.1:5432 (but docker postgres is running?)
```

### Expected Behaviour
- add pg-docker to README
- update example envs
- add git to Dockerfile

### CODE REVIEW

File: apps/onboarding/.env.local.example [hunk 1]
```diff
@@ -3,7 +3,13 @@ RELAYER_HOST=localhost
 RELAYER_PORT=5050
 DB_HOST=localhost
 DB_PORT=5432
-DB_USERNAME='postgres'
```
Comment (diminator) [inline apps/onboarding/.env.local.example:6]: maybe need to revert this?

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 FROM node:12-slim
-
+RUN apt update && apt install -y git
```
Comment (diminator) [inline Dockerfile:2]: maybe we can have a heavier node image that includes git

Comment (eruizgar91) [reply Dockerfile:2]: I am updating this in another PR

File: apps/onboarding/.env.local.example (hunk 1 above)
Comment (bowd) [reply apps/onboarding/.env.local.example:6]: Yeah we can leave the defaults in the example files as they were before. 
What I sent in slack was for my local config, and it was more in order to give you the rest of the configs.

**Review by bowd — APPROVED** [review]

**Review by eruizgar91 — APPROVED** [review]

**Pull request merged**